### PR TITLE
Prevent dependency packaging only sonarlint

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

**Comment by dbmeneses** [comment]
I'll try to test it tomorrow

File: src/main/java/org/sonarsource/pluginpackaging/SonarPluginMojo.java [hunk 1]
```diff
@@ -149,9 +153,19 @@ public File createArchive() throws MojoExecutionException {
...
-          "Packaging dependencies as resources is not supported anymore. Please set plugin property 'skipDependenciesPackaging' to 'true' and use the maven-shade-plugin to embed your plugin dependencies.");
+      if (isSkipDependenciesPackaging()) {
+        getLog().info("Skip packaging of dependencies");
+
+      } else {
+        List<String> libs = copyDependencies();
+        if (!libs.isEmpty()) {
+          if (isSonarLintSupported()) {
+            throw new MojoFailureException(
+              "A SonarLint compatible plugin should not package dependencies. Please set property 'skipDependenciesPackaging' to 'true' and use the maven-shade-plugin to shade instead.");
```
Comment (dbmeneses) [inline src/main/java/org/sonarsource/pluginpackaging/SonarPluginMojo.java:164]: Looks like this line is too long.

**Review by dbmeneses — APPROVED** [review]
Tested with sonarCobol.
The build is failing because of unauthorized access to repox. Maybe the credentials in travis are not up to date?

**Comment by henryju** [comment]
Yeah, I already contacted the RE team about that.

**Pull request merged**